sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntityMatch.cs

```diff
@@ -13,14 +13,18 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("Match")]
     public readonly partial struct LinkedEntityMatch
     {
+        // use for deserialization
         internal LinkedEntityMatch(double confidenceScore, string text, int offset, int length)
+            : this(confidenceScore, text, offset) => Length = length;
+
+        internal LinkedEntityMatch(double confidenceScore, string text, int offset)
         {
             // We shipped TA 5.0.0 Text == string.Empty if the service returned a null value for Text.
             // Because we don't want to introduce a breaking change, we are transforming that null to string.Empty
             Text = text ?? string.Empty;
             ConfidenceScore = confidenceScore;
             Offset = offset;
-            Length = length;
+            Length = default;
```

Comment:
why keeping `Length` in `LinkedEntityMatch`?

Reply:
I had to keep it because of how we are plugging codegen to this class. The property is private so no one can access it though.

Reply:
do you mean you codegenerate the LinkedEntityMatch to be explored to user?

Reply:
Also, see my comment to Shawn